platform/darwin/src/MGLCircleStyleLayer.h

```diff
@@ -6,13 +6,31 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+/**
+ Controls the translation reference point.
```

Comment:
Since the enums are on a different page than the properties that use them, perhaps this documentation could mention those properties.